classes/form/cms_types_form.php

```diff
@@ -52,6 +54,27 @@ protected function definition() {
         $mform->addElement('editor', 'description', get_string('description'));
         $mform->setType('description', PARAM_RAW);
 
+        $mform->addElement(
+            'textarea',
```

Comment:
can you also make this more like a code editor with some light css

font-family: monospace;
font-size: 12px;

direct inline css is probably fine

Reply:
done